### PR TITLE
858353-Removed the unwanted content from Sunburst UG.

### PR DESCRIPTION
**Pull request by NanthiniMahalingam**
**Description**
1. Removed the unwanted content from sunburst chart md file.
2. Modified the key feature content in over view md file.
3. Update the content review changes.
4. Added resized images.

### CODE REVIEW

File: MAUI/SunburstChart/Overview.md [hunk 1]
```diff
@@ -13,10 +13,7 @@ Syncfusion .NET MAUI Sunburst Chart is used to represent the hierarchical data s
...
-* User interaction: The sunburst chart's legend and tooltips enhance the end-user experience significantly.
-
-* The end-user experience is empowering for exploring hierarchical data, comprehending relationships, and making informed decisions by including features such as levels, data labels, and tooltip interaction.
-
-* Legends provide more information about the segments. If an item exceeds the available bounds, the legends can be scrolled.
-
-* Provides various options for customizing and visualizing the sunburst chart features, title, data labels, legends, tooltip, and etc.
+* Supports data labels and legends for better readability.
+* Supports tooltip to provide more information about the segments.
+* Can customize appearance with own palette brushes.
```
Comment (Saravanan-Madhesh) [inline MAUI/SunburstChart/Overview.md:18]: Can customize appearance with custom palette brushes.

Comment (NanthiniMahalingam) [reply MAUI/SunburstChart/Overview.md]: Updated the content sir

File: MAUI/SunburstChart/DataLabels.md [hunk 2]
```diff
@@ -8,10 +8,10 @@ documentation: ug
 ---
 
 # Data label in .NET MAUI Sunburst Chart
```
[the remark this answers is not in the document]
Comment (NanthiniMahalingam) [reply MAUI/SunburstChart/DataLabels.md]: Updated the title sir

**Review by Saravanan-Madhesh — APPROVED** [review]
Approved as got content review.

**Pull request merged**